migrations/20200927000000-add-indexes.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const {DataTypes} = Sequelize;
+    const transaction = await queryInterface.sequelize.transaction();
```

Comment:
Does this work? I thought you have to pass the transaction object to queries

```js
      queryInterface.changeColumn('gblcomment', 'content', {
        type: DataTypes.TEXT,
        allowNull: true
      }, { transaction });
```

Something like that?

Reply:
I honestly don't know I just followed stuff I found online.

A migration should use the same connection so in theory it should work